pkg/machine/provider/platform_darwin.go

```diff
@@ -28,10 +27,8 @@ func Get() (machine.VirtProvider, error) {
 
 	logrus.Debugf("Using Podman machine with `%s` virtualization provider", resolvedVMType.String())
 	switch resolvedVMType {
-	case define.QemuVirt:
-		return qemu.VirtualizationProvider(), nil
 	case define.AppleHvVirt:
-		return applehv.VirtualizationProvider(), nil
+		return new(applehv.AppleHVStubber), nil
```

Comment:
I don't love the "stubber" name but mostly a naming nit, not a big deal for now

Reply:
yeah, its unique, prob will leave it in until all the providers are in and then will implement a change.